iothub/service/src/Http/HttpMessageHelper.cs

```diff
@@ -95,7 +98,7 @@ internal static async Task<T> DeserializeResponseAsync<T>(HttpResponseMessage re
         /// </param>
         /// <exception cref="ArgumentNullException">Thrown when the provided <paramref name="requestMessage"/> or <paramref name="requestMessage"/> is null.</exception>
         /// <exception cref="ArgumentException">Thrown if the <paramref name="eTag"/> is empty or white space.</exception>
-        internal static void ConditionallyInsertETag(HttpRequestMessage requestMessage, ETag eTag, bool onlyIfUnchanged)
+        internal static void ConditionallyInsertETag(HttpRequestMessage requestMessage, ETag eTag, bool onlyIfUnchanged = false)
```

Comment:
Setting default to false intentional?

Reply:
This is weird, I am not aware of this, so changing it back.